plugins/doc_fragments/attributes.py

```diff
@@ -18,6 +18,20 @@ class ModuleDocFragment(object):
     description: Can run in C(check_mode) and return changed status prediction without modifying target.
   diff_mode:
     description: Will return details on what has changed (or possibly needs changing in C(check_mode)), when in diff mode.
+  idempotent:
+    description:
+      - When run twice in a row with the same arguments, the second invocation indicates no change.
```

Comment:
@russoz what do you think of the following clarification?
```suggestion
      - When run twice in a row with the same arguments outside check mode, the second invocation indicates no change.
```
While for some users it might be obvious that this generally doesn't work with check mode, some others might assume it to be the case.

Reply:
Maybe: 
```suggestion
      - When run twice in a row outside check mode, with the same arguments, the second invocation indicates no change.
```
Not much of a difference really, just feels slightly better to ~~my~~ the ears.